sway/tree/container.c

```diff
@@ -567,9 +567,8 @@ static struct sway_container *container_at_view(struct sway_container *swayc,
 		*sx = _sx;
 		*sy = _sy;
 		*surface = _surface;
-		return swayc;
 	}
-	return NULL;
+	return swayc;
```

Comment:
Hmm. So maybe we should make this function return `void` now, since it doesn't have the same semantics as other functions with the same signature?

Reply:
Hmm. I'd favour renaming the function to `surface_at_view` and then having it return void.

Reply:
Yeah, that seems sensible.